protocols/dcutr/src/behaviour_impl.rs

```diff
@@ -263,6 +266,7 @@ impl NetworkBehaviour for Behaviour {
             self.queued_events.extend([ToSwarm::GenerateEvent(
                 Event::DirectConnectionUpgradeSucceeded {
                     remote_peer_id: peer,
+                    connection_id: relayed_connection_id,
```

Comment:
This is not correct I think! :)

`connection_id` should be the direct one.

Reply:
Same with the other below.

Reply:
Nice catch!